File: app/src/main/java/edu/harding/pokemonteambuilder/Pokemon.java
package edu.harding.pokemonteambuilder;

import android.graphics.Bitmap;
import android.util.Pair;

import java.util.List;

public class Pokemon {

    private Bitmap mThumbnail;
    private String mName;
    private String mType;
    List<Move> mLearnedMoves;
    List<Pair<String, String>> mAbilities;

    public Bitmap getThumbnail() {
        return mThumbnail;
    }

    public void setThumbnail(Bitmap thumbnail) {
        this.mThumbnail = thumbnail;
    }

    public String getName() {
        return mName;
    }

    public void setName(String name) {
        this.mName = name;
    }

    public String getType() {
        return mType;
    }

    public void setType(String type) {
        this.mType = type;
    }

}

File: app/src/main/java/edu/harding/pokemonteambuilder/Pokemon.kt
package edu.harding.pokemonteambuilder

import android.graphics.Bitmap
import android.util.Pair

class Pokemon {

    var thumbnail: Bitmap? = null
    var name: String? = null
    var type: String? = null
    internal var mLearnedMoves: List<Move>? = null
    internal var mAbilities: List<Pair<String, String>>? = null

}
